e2e/utils/TestUtils.ts

```diff
@@ -157,8 +157,8 @@ export function reportAsset(testPath: string, type: 'trace' | 'log' = 'trace') {
 }
```

Comment:
Remove Cirrus from the note in line 155

Reply:
I'll keep that for now, until we wipe out `.cirrus.yml`; does that sound okay?